### ISSUE TITLE
IP Blacklisting

### ISSUE THREAD

**Issue by nathancani** [salesagility/SuiteCRM#653]
Feature sugggestion rather than Bug;

A feature needs to be added into suitecrm that blocks ip's from logging into the crm when an ip fails to login to the CRM after x amount of times; to avoid brute force logins to the crm

There is no configuration option that i can see for Native support without manually coding anything in.

**Comment by pgorod** [comment]
I would favor the fail2ban approach as explained here:
https://www.suitecrm.com/forum/suitecrm-7-0-discussion/1076-lock-user-account-after-x-login-attempts

We should avoid "reinventing the wheel". 

The fix to the code is incredibly simple, just make sure we log an error. Then fail2ban takes up the job from there.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [IP Blacklisting](https://trello.com/c/hOwicgNc/135-ip-blacklisting)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
